compiler/passes/scopeResolve.cpp

```diff
@@ -2404,6 +2404,19 @@ static void removeUnusedModules() {
   }
 }
 
+static void detectUserDefinedBorrowMethods() {
+  const char *astrBorrow = astr("borrow");
```

Comment:
Maybe use the global astrBorrow here too?

Reply:
oh, duh :(